site/src/components/segmented-controls.md

```diff
@@ -0,0 +1,266 @@
+					/>
+				</svg>
+			</span>
+		</button>
+
+		<div class="segmented-control-list" role="tablist" aria-orientation="horizontal">
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-one-1"
+				role="tab"
+				aria-selected="true"
+				data-segmented-control
+			>
+				Define Strategy
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-one-2"
+				role="tab"
+				data-segmented-control
+			>
+				Plan
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-one-3"
+				role="tab"
+				data-segmented-control
+			>
+				Prepare
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-one-4"
+				role="tab"
+				data-segmented-control
+			>
+				Adopt
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-one-5"
+				role="tab"
+				data-segmented-control
+			>
+				Govern
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-one-6"
+				role="tab"
+				data-segmented-control
+			>
+				Manage
+			</button>
+		</div>
+		<button
+			type="button"
+			class="segmented-control-next"
+			title="Next segment"
+			data-segmented-control-nav="next"
+		>
+			<span class="icon" aria-hidden="true">
+				<svg xmlns="http://www.w3.org/2000/svg" viewBox="-255 57 448 448">
+					<path
+						class="fill-current-color"
+						d="M-253.625 281v-32H131.75L-16.938 100.313 5.687 77.688 193 265 5.687 452.313l-22.625-22.625L131.75 281h-385.375z"
+					/>
+				</svg>
+			</span>
+		</button>
+	</div>
+
+	<div role="tabpanel" aria-labelledby="sc-one-1">
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 1</p>
+		</div>
+	</div>
+	<div role="tabpanel" aria-labelledby="sc-one-2" hidden>
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 2</p>
+		</div>
+	</div>
+	<div role="tabpanel" aria-labelledby="sc-one-3" hidden>
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 3</p>
+		</div>
+	</div>
+	<div role="tabpanel" aria-labelledby="sc-one-4" hidden>
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 4</p>
+		</div>
+	</div>
+	<div role="tabpanel" aria-labelledby="sc-one-5" hidden>
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 5</p>
+		</div>
+	</div>
+	<div role="tabpanel" aria-labelledby="sc-one-6" hidden>
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 6</p>
+		</div>
+	</div>
+</tab-container>
+```
+
+## Segmented Control with footer Next/Prev
+
+Removing the next and previous buttons will still provide a seamless experience on the tabs control. Small restructuring allows for the tabs-container component to allocate a footer to show after the tabpanel. The key difference is that of the `div.segmented-controls` taking on the `role=tablist` and `aria-orientation` attributes. This restructuring provides the grid responsiveness without buttons next to the control-list.
```

Comment:
Docs are mixing styling with web component again.

Reply:
Going to remove the footer nav altogether.   I will be immediately revisiting this and working towards a cleaner presentation and approach.  Might include some adjustments to the tab-container source code because its not accounting for things properly on the after panels.  So keep an eye out for that PR follow up.